### PR TITLE
Update content box

### PR DESCRIPTION
**Pull request by aju**
I added it so we know that this functionality is needed when designing new system

### CODE REVIEW

File: src/components/content-box/ContentBox.jsx [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+// @flow
+import * as React from 'react';
 import classNames from 'classnames';
 import {SIZE} from './ContentBoxConstants';
 
-const ContentBox = ({children, spacedTop, spacedBottom, spaced, full, className, ...props}) => {
+export type ContentBoxPropsType = {
+  children: React.Node,
+  spaced?: boolean,
+  spacedSmall?: boolean,
```
Comment (vergilius) [inline src/components/content-box/ContentBox.jsx]: this is new, right?

Comment (aju) [reply src/components/content-box/ContentBox.jsx]: yes

File: src/components/content-box/ContentBoxConstants.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// @flow
 const SIZE = {
```
Comment (burakukula) [inline src/components/content-box/ContentBoxConstants.jsx]: maybe `Object.freeze` here?

**Comment by burakukula** [comment]
@aju you could try to bump SG from a branch in brainly-frontend, to check if there is a lot of changes if so to revert flow before you actually push new SG tag.

**Pull request merged**